test/Test/Ouroboros/Network/Node.hs

```diff
@@ -99,15 +100,20 @@ test_blockGenerator chain slotDuration = isValid <$> withProbe (experiment slotD
       :: ( MonadSTM m
          , MonadTimer m
          , MonadProbe m
+         , MonadSay m
```

Comment:
Do we need this extra constraints? I bet it's a debugging leftover.

Reply:
Sure... the lefties after debugging!